src/knx/cemi_frame.h

```diff
@@ -62,4 +75,9 @@ class CemiFrame
     TPDU _tpdu;
     APDU _apdu;
     uint16_t _length = 0; // only set if created from byte array
+
+    // nly for RF medium
+    uint8_t* _rfSerialOrDoA = 0;
```

Comment:
Are they encoded in the data somewhere or is this some kind of additional info not included in the actual cemi-frame?

Reply:
You are right. This is contained in the RF additional info. 
However, it does not make sense to artificially create an RF info additonal info structure and add it to the buffer just to let it parse again in the cEMI frame class. The specification also says that for local usage in the stack it is not necessary to use additonal info. It is really only meant for messages which are leaving one device and enter another one (USB RF stick for example).
So I would like keep it there as is.
Should someone implement the USB HID KNX data link layer, the cEMI class would need refactoring anyway so that it is able to parse additional infos.

Reply:
Ok.

Reply:
nothing done.